kaitaistruct.py

```diff
@@ -451,19 +453,19 @@ class ValidationLessThanError(ValidationFailedError):
     """Signals validation failure: we required "actual" value to be
     greater than or equal to "min", but it turned out that it's not.
     """
-    def __init__(self, min, actual, io, src_path):
-        super(ValidationLessThanError, self).__init__("not in range, min %s, but got %s" % (repr(min), repr(actual)), io, src_path)
-        self.min = min
+    def __init__(self, minBound, actual, io, src_path):
```

Comment:
I don't like the new name. The old `min` is clear enough.

And BTW, you have to keep the naming convention consistent all the time. You just can't have two semantically equivalent identifiers `minBound` and `src_path` on the same line. Use the `snake_case` for parameter names as the rest of the code does.